src/api/MarkerBasedAPI.js

```diff
@@ -62,6 +62,7 @@ class MarkerBasedApi extends Base {
                 ...requestData,
                 marker,
                 limit,
+                usemarker: true,
```

Comment:
just confirming - this change won't impact any other APIs that use this MarkerBasedAPI? i.e. if they don't have this query param implemented, it'll just be ignored?

Reply:
in most cases, we use camelCase, but this one is all lowercase. Is that what we have to use here?

Reply:
Good question. The `usemarker` parameter enforces the use of marker-based paging when both offset-based paging and marker-based paging are available. It shouldn't affect the other files that use this API, but to be safe, I'll move this parameter to the `MarkerBasedGroups` and `MarkerBasedUsers` files.

Reply:
@booc0mtaco Yeah, this parameter is specified as all lowercase in the docs. I was surprised by the format, too.